indra/newview/skins/default/xui/en/panel_preferences_game_control.xml

```diff
@@ -10,198 +10,206 @@
         control_name="GameControlToAgent"
-        follows="top|left"
-        height="15"
         label="GameControl moves avatar and flycam"
         layout="topleft"
+        height="15"
         left="10"
-        name="game_control_to_agent"
-        top="27"
-        width="232"/>
+        top="30"/>
     <check_box
+        name="agent_to_game_control"
         control_name="AgentToGameControl"
-        follows="top|left"
-        height="15"
         label="Avatar actions interpreted as GameControl"
         layout="topleft"
+        height="15"
         left="10"
-        name="agent_to_game_control"
-        top="48"
-        width="232"/>
-    <scroll_list
-        draw_heading="true"
-        follows="all"
+        top="50"/>
+    <tab_container
+        name="game_control_tabs"
         layout="topleft"
-        column_padding="0"
-        selection_type="header"
-        top="66"
-        left="3"
-        bottom="-3"
-        right="-3"
-        can_sort="false"
-        multi_select="false"
-        name="action_table"
-        fg_disable_color="ScrollUnselectedColor"/>
+        follows="all"
+        top="70"
+        left="2"
+        right="-2"
+        bottom="-2">
+        <panel label="Channel mappings">
+            <scroll_list
+                name="action_table"
+                layout="topleft"
+                follows="all"
+                top="1"
+                left="1"
+                right="-1"
+                bottom="-1"
+                can_sort="false"
+                column_padding="0"
+                multi_select="false"
+                selection_type="header"
+                fg_disable_color="ScrollUnselectedColor">
+                <scroll_list.columns
+                    name="action"
+                    label="Action"
+                    relative_width="0.25" />
+                <scroll_list.columns
+                    label="Channel"
+                    relative_width="0.75" />
+            </scroll_list>
+        </panel>
+        <panel label="Device settings">
+        </panel>
+    </tab_container>
     <combo_box
-        height="23"
-        layout="topleft"
-        left="10"
         name="analog_channel_selector"
-        top_pad="5"
-        width="90">
+        width="380"
+        height="18">
         <combo_box.item
-            label="AXIS_0-"
+            label="AXIS_0- (Left stick: push left)"
             name="AXIS_0-"
             value="AXIS_0-"/>
         <combo_box.item
-            label="AXIS_0+"
+            label="AXIS_0+ (Left stick: push right)"
             name="AXIS_0+"
             value="AXIS_0+"/>
         <combo_box.item
-            label="AXIS_1-"
+            label="AXIS_1- (Left stick: push up)"
             name="AXIS_1-"
             value="AXIS_1-"/>
         <combo_box.item
-            label="AXIS_1+"
+            label="AXIS_1+ (Left stick: push down)"
             name="AXIS_1+"
             value="AXIS_1+"/>
```

Comment:
Up is positive and down is negative, so swap the two descriptions here, and also for the up-down cases below.